src/DevLicenseCore.sol

```diff
@@ -160,6 +164,16 @@ contract DevLicenseCore is IDevLicenseDimo, AccessControl {
         _disableSigner(tokenId, signer);
     }
 
+    /**
+     * @notice It sets an alias to a token ID
+     * @dev Only the token ID owner can call this function
+     * @param tokenId The unique identifier for the license token.
+     * @param licenseAlias The alias string to be set
+     */
+    function setLicenseAlias(uint256 tokenId, bytes32 licenseAlias) public onlyTokenOwner(tokenId) {
```

Comment:
What's the convention here? Is it supposed to be a short string padded with a bunch of spaces at the end? A hash?

Reply:
Ah, they null terminate. I would comment on this, it's a bit different.

Reply:
Is it a feature or a danger that you can "take over" someone's old alias?

Reply:
It is supposed to be anything as long as it fits in bytes32. Theorically we could have 2**256-1 licenses, an alias for each one \o/
It kinda works similar to ENS/DCN, as long as you have the NFT, no one else can take it
I don't know if I want to add too much complexity to those alias

Reply:
I wouldn't ask for more.